server.js

```diff
@@ -0,0 +1,5 @@
+const express = require('express');
+let app = express()
+app.listen(8008)
+app.use(express.static('public'));
+app.use(express.static('node_modules'));     
```

Comment:
use `__dirname + '/public'` (and same for node_modules) - this guarantees our path is what we want it to be (important when we'll actually host our websites off our computer